docs/installation.rst

```diff
@@ -2,61 +2,63 @@
 Requirements and Installation
 *****************************
 
-Ubuntu 18.04 
+Ubuntu 18.04
 ~~~~~~~~~~~~
 
-CubiCal depends on python-casacore, the dependencies of which should be 
-installed from the KERN-5 ppa. Note that ``apt-get install`` is ``apt install``
-on 18.04.
+CubiCal depends on python-casacore, the dependencies of which should be
+installed from the KERN-5_ ppa. The ppa can be added as follows:
 
-.. code:: bash
+.. _KERN-5: https://kernsuite.info/installation/
 
-	ENV DEB_DEPENDENCIES casacore-dev \
-                     casacore-data \
-                     build-essential \
-                     python3-pip \ 
-                     libboost-all-dev \ 
-                     wcslib-dev \
-                     git \
-                     libcfitsio-dev
-	apt-get update
-	apt-get install -y $DEB_DEPENDENCIES
-	pip3 install -U pip wheel setuptools
-	python3.6 -m pip install -U .
-	python3.6 -m pip install path/to/repo/
-
-If you wish to install CubiCal in a virtual environment (recommended), see 
-`Using a virtual environment`_. 
-
-.. note:: 
-
-	At this point, if CubiCal is required to predict model visiblities, it is necessary 
-	to install Montblanc. The CPU version of montblanc is installed automatically if montblanc has not been previously
-	installed. To install the GPU version of Montblanc, follow the instructions here_ before installing cubical.
+.. code:: bash
 
-	.. _here: https://montblanc.readthedocs.io
+	sudo apt install software-properties-common
+	sudo add-apt-repository -s ppa:kernsuite/kern-5
+	sudo apt-add-repository multiverse
+	sudo apt-add-repository restricted
+	sudo apt update
 
-CubiCal can now be installed from PyPI by running the following:
+Once the ppa has been added, CubiCal's dependencies can be installed as
+follows:
 
 .. code:: bash
 
-	pip install cubical
+	CUBICAL_DEPENDENCIES=(casacore-dev \
+                     	      casacore-data \
+                              build-essential \
+                              python3-pip \
+                              libboost-all-dev \
+                              wcslib-dev \
+                              git \
+                              libcfitsio-dev)
+	sudo apt install -y $CUBICAL_DEPENDENCIES
 
-.. warning:: 
+If you wish to install CubiCal in a virtual environment (recommended), see
+`Using a virtual environment`_.
 
-	To install in development mode, you will need to install some dependencies manually
-	and cythonize the development kernels explicitly. Assuming that you have already
-	cloned the repository, this can be done as follows:
+.. note::
 
-	.. code:: bash
+	CubiCal predicts model visiblities using Montblanc_. The CPU version of Montblanc is
+	installed automatically. To install the GPU version of Montblanc, follow the
+	instructions here_ before installing cubical.
+
+	.. _Montblanc: https://arxiv.org/abs/1501.07719
+	.. _here: https://montblanc.readthedocs.io
```

Comment:
These instructions are out of date. Use the setup.py optional dependencies

Reply:
specifically cubical does not work with the master branch

Reply:
I have updated the installation instructions to include the optional Montblanc dependency.